OpenMEEG/libs/OpenMEEG/include/integrator.h

```diff
@@ -177,7 +177,7 @@ namespace OpenMEEG {
 
     static const unsigned nbPts[4] = {3, 6, 7, 16};
 
-    template <class T, class I>
```

Comment:
actually it is because when using openblas, complex.h is included, so the compiler complains (because I is not a complex imaginary...) hence the fix.
I've updated with typename this time